### ISSUE TITLE
Assignment 10

### ISSUE THREAD

**Issue by twhitacre** [tiy-atl-js-sept-2015/Assignments#266]
@amitshah86 this is your issue for *Assignment 10*. Please comment below with the required deliverable(s) and close this issue when you are finished.

**Comment by amitshah86** [comment]
http://amitshah86.github.io/Assignment10

**Comment by jisaacks** [comment]
It is not currently working. I think maybe when you wrote `btn.addClass('h2.box3');` you mean `btn.addClass('box3');` etc. That might help. Please resubmit once you get it working. Thanks :)

**Comment by amitshah86** [comment]
http://amitshah86.github.io/Assignment10.

Okay i think ive got it figured out.

**Comment by jisaacks** [comment]
I see you made changes in your master branch but it looks like you forgot to push them to gh-pages.

**Comment by amitshah86** [comment]
let me try again...it kept saying that everything is up to date

**Comment by jisaacks** [comment]
Did you merge master into gh-pages before trying to push gh-pages?

**Comment by amitshah86** [comment]
http://amitshah86.github.io/Assignment10

**Comment by jisaacks** [comment]
You are getting close, it is functional. There still is no transitioning happening though, try to get that working.

**Comment by amitshah86** [comment]
Okay I'll do it now. I know what I have to change.
